projects/dsp-ui/src/lib/search/expert-search/expert-search.component.ts

```diff
@@ -82,8 +85,13 @@ CONSTRUCT {
     submitQuery() {
         const gravsearch = this._generateGravsearch(0);
 
+        // const gravsearchWithoutOffset = this.expertSearchForm.controls['gravsearchquery'].value;
```

Comment:
If this line is not used, it should be removed otherwise a todo comment should be written.

Reply:
Done in f0268ac